src/styles/partials/components/_card-container.scss

```diff
@@ -0,0 +1,26 @@
+.dg_card {
+  border: $border-width-small solid $gray;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+  margin-bottom: $margin-large;
+  padding: $padding-large;
+  width: 100%;
+
+  .dg_icon-container {
+    color: $gold;
+    font-size: 60px;
+    margin-bottom: 20px;
+  }
+
+  .dg_card__content {
```

Comment:
I think if this is the default style you would just need to add text-align: center to dg_card and remove lines 18-25?

Reply:
Removed lines and committed.